yazi-config/src/manager/mouse.rs

```diff
@@ -45,7 +45,7 @@ impl From<MouseEvents> for Vec<String> {
 			(MouseEvents::MOVE, "move"),
 			(MouseEvents::DRAG, "drag"),
 		];
-		events.into_iter().filter(|v| value.contains(v.0)).map(|v| v.1.to_owned()).collect()
+		events.into_iter().filter(|(e, _)| value.contains(*e)).map(|(_, e)| e.to_owned()).collect()
```

Comment:
I don't like this style because it has two `e`s at the same time, which makes it less recognizable compared to the previous `v.0` and `v.1`.

Reply:
How about this one?
```rs 
events.into_iter().filter(|(a, _)| value.contains(*a)).map(|(_, b)| b.to_owned()).collect()
```

Reply:
IMO this doesn't make it clearer, since when I see `a` and `b`, I still have to think about what corresponds to `v.0` and `v.1`, so why not just use `v.0` and `v.1` directly?